docs/en/faq.md

```diff
@@ -150,9 +150,35 @@ We list some common troubles faced by many users and their corresponding solutio
 - "GPU out of memory"
 
   1. There are some scenarios when there are large amount of ground truth boxes, which may cause OOM during target assignment. You can set `gpu_assign_thr=N` in the config of assigner thus the assigner will calculate box overlaps through CPU when there are more than N GT boxes.
+
   2. Set `with_cp=True` in the backbone. This uses the sublinear strategy in PyTorch to reduce GPU memory cost in the backbone.
+
   3. Try mixed precision training using following the examples in `config/fp16`. The `loss_scale` might need further tuning for different models.
 
+  4. Try to use `AvoidOOM` to avoid GPU out of memory. It will first retry after calling `torch.cuda.empty_cache()`. If it still fails, it will then retry by converting the type of inputs to FP16 format. If it still fails, it will try to copy inputs from GPUs to CPUs to continue computing. Try AvoidOOM in you code to make the code continue to run when GPU memory runs out:
+
+     ```python
+     # To use as a function
+     from mmdet.utils import AvoidOOM
+
+     AvoidCUDAOOM = AvoidOOM()
+     # GPU OOM error
+     # outputs = some_torch_function(input1, input2)
+     output = AvoidCUDAOOM.retry_if_cuda_oom(some_torch_function)(input1, input2)
+     ```
+
```

Comment:
May also add some description in between the two usages.

Reply:
I think we should simply use AvoidCUDAOOM here, which squeeze the code into two lines:
```
from mmdet.utils import AvoidCUDAOOM

output = AvoidCUDAOOM.retry_if_cuda_oom(some_torch_function)(input1, input2)
```